src/Draft/Attachments/AudioPlayer.vue

```diff
@@ -1,16 +1,30 @@
 <template>
-  <audio v-if="url != null" controls>
+  <audio
+    v-if="url != null"
+    ref="audioElement"
+    controls
+    @loadedmetadata="onLoadedMetadata"
```

Comment:
oh wow! this is clever. I would not have thought about getting it from the player.